python/python/tests/tests_server_modules/test_json.py

```diff
@@ -1954,3 +1960,21 @@ async def test_json_arrpop(self, glide_client: TGlideClient):
 
         assert await json.arrpop(glide_client, key2, JsonArrPopOptions("[*]")) == b'"a"'
         assert await json.get(glide_client, key2, ".") == b'[[],[],["a"],["a","b"]]'
+
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_json_transaction(self, glide_client: GlideClusterClient):
+        transaction = ClusterTransaction()
+
```

Comment:
Is adding tests for the remaining commands a separate task?

Reply:
My plan is to expand testing for remaining commands as a separated task.

Reply:
we should add tests at the same time.